### PR TITLE
Tail grabbing rising: revengeance (nerfed)

### PR DESCRIPTION
**Pull request by Kapu1178**
## About The Pull Request
**PERMISSION: TheFakeElon**

Grabbing the groin region of anyone with a cat tail now has a custom message, and applies the effects of Grab Weakness (If resting, resisting out of a grab is equivalent to 1 grab state higher. wont make the grab state exceed the normal max, however). 

If people want I can apply this to all tails, I just found it extra funny it only applies to cat tails.

If you stop targeting the groin, it will stop applying grab weakness.
**Note: I plan on adding a new felinid ability soon-ish**

## Why It's Good For The Game
Felinid table efficacy increases 200%, and is funny.

## Changelog
:cl:
balance: If you have a cat tail and are tail grabbed, you have grab weakness
/:cl:

### CODE REVIEW

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -293,9 +293,14 @@
 
 		log_combat(src, M, "grabbed", addition="passive grab")
 		if(!supress_message && !(iscarbon(AM) && HAS_TRAIT(src, TRAIT_STRONG_GRABBER)))
-			M.visible_message("<span class='warning'>[src] grabs [M] [(zone_selected == "l_arm" || zone_selected == "r_arm")? "by their hands":"passively"]!</span>", \
-							"<span class='warning'>[src] grabs you [(zone_selected == "l_arm" || zone_selected == "r_arm")? "by your hands":"passively"]!</span>", null, null, src)
-			to_chat(src, "<span class='notice'>You grab [M] [(zone_selected == "l_arm" || zone_selected == "r_arm")? "by their hands":"passively"]!</span>")
+			var/mob/living/L = M
+			if (L.getorgan(/obj/item/organ/tail/cat) && zone_selected == BODY_ZONE_PRECISE_GROIN)
+				M.visible_message("<span class ='warning'> [src] grabs [L] by the tail!</span>", null, null, src)
+				to_chat(src, "<span class='notice'>You grab [L] by their tail! Weirdo!</span>")
```
Comment (PowerfulBacon) [inline code/modules/mob/living/living.dm:299]: This should be merged into 1 line and use the proper functionality of visible_message

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
1. That procs getting kind of messy
2. This is literally just the weak grabber trait.
If its just going to be the weak grabber trait then
 A) Why not make felinids just have the weak grabber trait
 B) Felinids now have no downside to taking that trait so get free points.
3. Also this is a dumb PR like everything that touches felinids

**Comment by Kapu1178** [comment]
> 
> 
>     1. That procs getting kind of messy
> 
>     2. This is literally just the weak grabber trait.
>        If its just going to be the weak grabber trait then
>        A) Why not make felinids just have the weak grabber trait
>        B) Felinids now have no downside to taking that trait so get free points.
> 
>     3. Also this is a dumb PR like everything that touches felinids

1. I can put alot of comments if you want to make it clear exactly what everything does

2. Yes, but only under the condition you grab someone's tail. Its a funny little flavor bit that realistically very few people if anyone will exploit. At worst I can just remove it all together if it really is a problem, which I dont think it is.

3. Other people like it.

File: code/modules/mob/living/living.dm (hunk 1 above)
Comment (Kapu1178) [reply code/modules/mob/living/living.dm:299]: Tried, doesnt work. Did add a missing string, though

**Pull request closed without merging**